keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -107,8 +107,12 @@ object will be used otherwise (e.g. `pods/proxy`) the labels on the existing par
 object will be used. If subresource does not have a parent object, the Webhook call 
 may fail or skipped based on the failure policy.
 
-When deleting a collection, the webhook will get a filtered list of objects if the 
-ObjectSelector is applied. The webhook may be called multiple times with sub-lists.
+When deleting a collection, the webhook gets called for each object matching the
+ObjectSelector. Post GA, if we find the performance cost of sending O(#objects)
```

Comment:
I'd omit the forward-looking statements from this doc. Knowing we have a way to opt into batching in the future is enough for me.